src/main/java/Duke.java

```diff
@@ -19,20 +21,90 @@ public static void main(String[] args) {
+                    continue;
+                }
+
+                String description = userInput.substring(5).trim();
+                Todo newTodo = new Todo(description);
+                tasks[taskCount] = newTodo;
+                taskCount++;
+
+                System.out.println("____________________________________________________________");
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  " + newTodo);
+                System.out.println("Now you have " + taskCount + " tasks in the list.");
+                System.out.println("____________________________________________________________");
+            } else if (userInput.startsWith("deadline ")) {
+                if (taskCount >= 100) {
+                    System.out.println("You have reached the task limit. Cannot add more tasks!");
+                    continue;
+                }
+
+                String content = userInput.substring(9).trim();
+                int index = content.indexOf("/by");
+
+                if (index != -1) {
+                    System.out.println("Please use '/by' to specify the deadline time.");
+                    String description = content.substring(0, index).trim();
+                    String by = content.substring(index + 4).trim();
+                    Deadline newDeadline = new Deadline(description, by);
+                    tasks[taskCount] = newDeadline;
+                    taskCount++;
+                    System.out.println("____________________________________________________________");
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println("  " + newDeadline);
+                    System.out.println("Now you have " + taskCount + " tasks in the list.");
+                    System.out.println("____________________________________________________________");
+
+                } else {
+                    System.out.println("Please use '/by' to specify the deadline time.");
+                }
+            } else if (userInput.startsWith("event ")) {
+                if (taskCount >= 100) {
+                    System.out.println("You have reached the task limit. Cannot add more tasks!");
+                    continue;
+                }
+
+                String content = userInput.substring(6).trim();
+
+                String[] parts = content.split("/from | /to ");
+
+                if (parts.length < 3) {
+                    System.out.println("Please use the format: event [description] /from [start time] /to [end time]");
+                    continue;
+                }
+
+                String description = parts[0].trim();
+                String from = parts[1].trim();
+                String to = parts[2].trim();
+
+                Event newEvent = new Event(description, from, to);
+                tasks[taskCount] = newEvent;
+                taskCount++;
+
```

Comment:
In my opinion, here is a bit long nesting. Some conditions can be separated out as an isolated else if statement.